src/tests/ftest/soak/faults.yaml

```diff
@@ -124,9 +124,7 @@ ior_faults:
     - '1m'
   segment_count: 1
   dfs_oclass:
-    - 'RP_2G1'
-  dfs_dir_oclass:
-    - 'RP_2G1'
+    - ["EC_2P1GX", "RP_2GX"]
```

Comment:
FYI that ior doesn't use `--dfs.dir_oclass`, even if passed in, because IOR only works with files, not directories